### PR TITLE
add DesignPrinciples.md

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**

### CODE REVIEW

**Review by ttngu207 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
 <dd>  a collection of curated modules for assembling workflows for the major modalities of neurophysiology experiments + related documentation, tools, and utilities.
 </dl>
 
+## Elements
+* Lab management https://github.com/datajoint/element-lab
```
Comment (kabilar) [inline README.md:32]:
```suggestion
* Lab management https://github.com/datajoint/elements-lab 
```

File: README.md [hunk 2]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
 <dd>  a collection of curated modules for assembling workflows for the major modalities of neurophysiology experiments + related documentation, tools, and utilities.
 </dl>
 
+## Elements
+* Lab management https://github.com/datajoint/element-lab 
+* Animal management https://github.com/datajoint/element-animal
```
Comment (kabilar) [inline README.md:33]:
```suggestion
* Animal management https://github.com/datajoint/elements-animal
```

File: README.md [hunk 3]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
+  * Example workflow  https://github.com/datajoint/worklow-animal
```
Comment (kabilar) [inline README.md:34]:
```suggestion
  * Example workflow  https://github.com/datajoint/workflow-animal
```

File: README.md [hunk 4]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
+* Experiment session https://github.com/datajoint/element-session
```
Comment (kabilar) [inline README.md:35]:
```suggestion
* Experiment session https://github.com/datajoint/elements-session
```

File: README.md [hunk 5]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
+* Extracellular array electrophysiology https://github.com/datajoint/element-array-ephys
```
Comment (kabilar) [inline README.md:36]:
```suggestion
* Extracellular array electrophysiology https://github.com/datajoint/elements-array-ephys
```

File: README.md [hunk 6]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
+  * Example workflow https://github.com/datajoint/wokrlow-array-ephys
```
Comment (kabilar) [inline README.md:37]:
```suggestion
  * Example workflow https://github.com/datajoint/workflow-array-ephys
```

File: README.md [hunk 7]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
+* Calcium imaging https://github.com/datajoint/element-calcium-imaging
+  * Example workflow https://github.com/datajoint/workflow-calcium-imaging
+* Miniscope imaging https://github.com/datajoint/element-miniscope
```
Comment (kabilar) [inline README.md:40]:
```suggestion
* Miniscope imaging https://github.com/datajoint/elements-miniscope
```

File: README.md [hunk 8]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
 </dl>
 
+## Elements
+* Lab management https://github.com/datajoint/element-lab 
+* Animal management https://github.com/datajoint/element-animal
+  * Example workflow  https://github.com/datajoint/worklow-animal
+* Experiment session https://github.com/datajoint/element-session
+* Extracellular array electrophysiology https://github.com/datajoint/element-array-ephys
+  * Example workflow https://github.com/datajoint/wokrlow-array-ephys
+* Calcium imaging https://github.com/datajoint/element-calcium-imaging
```
Comment (kabilar) [inline README.md:38]:
```suggestion
* Calcium imaging https://github.com/datajoint/elements-calcium-imaging
```

File: README.md [hunk 9]
```diff
@@ -27,6 +28,23 @@ The Resource provides the following main components:
...
+  * Example workflow https://github.com/datajoint/workflow-calcium-imaging
+* Miniscope imaging https://github.com/datajoint/element-miniscope
+  * Example workflow https://github.com/datajoint/workflow-miniscope
+   
+## Example workflows 
+* Electrophysiology workflow with lab and animal management: https://github.com/datajoint/workflow-ephys
+* Imaging workflow with lab and animal management https://github.com/datajoint/workflow-imaging
```
Comment (kabilar) [inline README.md:45]:
```suggestion
```

File: DesignPrinciples.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
+# Design Principles 
+
+The following convnetions describe the Python implementation. Matlab conventions are similar and will be described separately.
```
Comment (kabilar) [inline DesignPrinciples.md:3]:
```suggestion
The following conventions describe the Python implementation. Matlab conventions are similar and will be described separately.
```

**Review by kabilar — APPROVED** [review]
I modified `element` from singular to plural, before I knew of the plan to rename the packages.  I will update in a subsequent pull request.

**Pull request merged**